tutorial/index.jade

```diff
@@ -62,18 +63,83 @@
       [workers-doc]: https://www.twilio.com/docs/api/taskrouter/workers
       [activities-doc]: https://www.twilio.com/docs/api/taskrouter/activities
 
+  .step(data-title='The Create Workspace Command', data-file='task_router/management/commands/create_workspace.py',
```

Comment:
@shyba I would try to show workspace.py here. Not sure how, but it might be more valuable to show the calls to Twilio's methods. Perhaps another step.